dev_tools/configure_vscode_for_bazel.py

```diff
@@ -192,13 +243,42 @@ def get_new_launch_config(executable_labels: set[str]) -> dict[str, Any]:
                     "binary_path": get_path_from_label(label),
                     "generated_by": "configure-vscode-for-bazel",
                 },
+                **try_get_task_dependency(label, task_exists=build_tasks_exist),
```

Comment:
`try` in my mind suggests a potential failure. This never fails, I vote for removing the `try_` prefix.